openmetadata-service/pom.xml

```diff
@@ -405,6 +405,36 @@
       <artifactId>unboundid-ldapsdk</artifactId>
       <version>${unboundsdk.version}</version>
     </dependency>
+
+    <!-- Argo Workflows -->
+    <dependency>
+      <groupId>io.argoproj.workflow</groupId>
+      <artifactId>argo-client-java</artifactId>
+      <version>v3.4.2</version>
+      <scope>system</scope>
+      <systemPath>${project.basedir}/src/main/resources/pipelineServiceClient/argo-client-java-v3.4.2.jar</systemPath>
```

Comment:
the thought process of adding this to the resources directory is that the jar is not available in mavenrepo. Instead, argoproj publishes it as a Github package, and retrieving that through pom and maven repositories requires authenticating to github.

to ease the setup process and do not add noise to the CI, adding the jar locally seems a sensible middle ground

Reply:
Have we run Snyk after this change to check they don't have any security risk?